app/templates/components/filter-nav.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="nav-3">
+  <div class="filter-by" {{action "touchEnd" }}>
+    <span>Filter by</span>
+    <p>All Employees</p>
+    <img src="/assets/images/accordion-open.svg" alt="img" class='open {{if clicked "turn"}}'>
+  </div>
```

Comment:
Use bs-dropdown and change this logic as per the dropdown property.

#dd.toggle--> All employee text and img
#dd.menu --> Move the filet-by component changes here.

Reply:
I have fixed this change and raised in next PR.